tests/integration/test_pytorch_lightning_model_artifact.py

```diff
@@ -0,0 +1,42 @@
+import torch
+import pandas as pd
+
+
+import bentoml
+from bentoml.utils.tempdir import TempDirectory
+from tests.bento_service_examples.pytorch_lightning_classifier import (
+    PytorchLightningService,
+)
+from tests.integration.api_server.conftest import export_service_bundle
```

Comment:
Better to import from here?
https://github.com/bentoml/BentoML/blob/master/tests/integration/utils.py#L40